mobile/src/libs/components/header/header.tsx

```diff
@@ -20,13 +20,15 @@ type Properties = {
   isArrowVisible?: boolean;
   badgeCount?: number;
   isProfileVisible?: boolean;
+  fontSize?: 'small' | 'large';
 };
 
 const Header: React.FC<Properties> = ({
   title,
   isArrowVisible = false,
   badgeCount = DEFAULT_BADGE_COUNT,
   isProfileVisible = false,
+  fontSize,
```

Comment:
let's make it large by default and remove from components where it's used
```suggestion
  fontSize = 'large',
```